kbase-extension/scss/partials/_job-status.scss

```diff
@@ -0,0 +1,125 @@
+.kb-job-status {
+  box-shadow: none;
+
+  &__row {
+    .show_log {
+      display: none;
+    }
+
+    .selected_log {
+      display: none;
+    }
+
+    &:hover {
+      background: #F2EFEB;
```

Comment:
I think all the colors here are included in the `_colours.scss` module. It would be best to try to stick with importing those.

Reply:
Fixed!